src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerSourceConfig.java

```diff
@@ -84,10 +84,6 @@ public void validate(FailureCollector collector) {
     if (!containsMacro(NAME_SCHEMA) && schema != null) {
       SpannerUtil.validateSchema(schema, SUPPORTED_TYPES, collector);
     }
-    if (!containsMacro(NAME_SCHEMA) && schema == null) {
```

Comment:
this will undo the changes for https://cdap.atlassian.net/browse/PLUGIN-251
Correct fix is to do something similar to bigquery source:
https://github.com/data-integrations/google-cloud/blob/develop/src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQuerySource.java#L194

Reply:
updated